### ISSUE TITLE
500 error when removing grades

### ISSUE THREAD

**Issue by pdpinch** [mitodl/edx-sga#144]
We got a number of these on the MITx residential stack (running cypress) yesterday:

```
Traceback (most recent call last):

 File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 109, in get_response
   response = callback(request, *callback_args, **callback_kwargs)

 File "/edx/app/edxapp/edx-platform/lms/djangoapps/courseware/module_render.py", line 854, in handle_xblock_callback
   return _invoke_xblock_handler(request, course_id, usage_id, handler, suffix, course=course)

 File "/edx/app/edxapp/edx-platform/lms/djangoapps/courseware/module_render.py", line 974, in _invoke_xblock_handler
   resp = instance.handle(handler, req, suffix)

 File "/edx/app/edxapp/venvs/edxapp/src/xblock/xblock/mixins.py", line 86, in handle
   return self.runtime.handle(self, handler_name, request, suffix)

 File "/edx/app/edxapp/edx-platform/common/lib/xmodule/xmodule/x_module.py", line 1269, in handle
   return super(MetricsMixin, self).handle(block, handler_name, request, suffix=suffix)

 File "/edx/app/edxapp/venvs/edxapp/src/xblock/xblock/runtime.py", line 1025, in handle
   results = handler(request, suffix)

 File "/edx/app/edxapp/venvs/edxapp/src/edx-sga/edx_sga/sga.py", line 446, in remove_grade
   module = StudentModule.objects.get(pk=request.params['module_id'])

 File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/db/models/manager.py", line 131, in get
   return self.get_query_set().get(*args, **kwargs)

 File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/db/models/query.py", line 358, in get
   clone = self.filter(*args, **kwargs)

 File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/db/models/query.py", line 624, in filter
   return self._filter_or_exclude(False, *args, **kwargs)

 File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/db/models/query.py", line 642, in _filter_or_exclude
   clone.query.add_q(Q(*args, **kwargs))

 File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/db/models/sql/query.py", line 1250, in add_q
   can_reuse=used_aliases, force_having=force_having)

 File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/db/models/sql/query.py", line 1185, in add_filter
   connector)

 File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/db/models/sql/where.py", line 69, in add
   value = obj.prepare(lookup_type, value)

 File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/db/models/sql/where.py", line 320, in prepare
   return self.field.get_prep_lookup(lookup_type, value)

 File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/db/models/fields/__init__.py", line 310, in get_prep_lookup
   return self.get_prep_value(value)

 File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/db/models/fields/__init__.py", line 537, in get_prep_value
   return int(value)

ValueError: invalid literal for int() with base 10: 'undefined'
```

**Comment by pdpinch** [comment]
@amir-qayyum-khan can you look to see if you can reproduce this -- either in dogwood or cypress?

**Comment by renankataoka** [comment]
I have the same issue with Eucalyptus 2. Did anyone find a fix for this?

**Comment by pdpinch** [comment]
fixed in v0.6.2 via 7e26f45ee3377aaea7fd4c4bb9797a78c0ccd106
